new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java

```diff
@@ -351,8 +349,7 @@ public static boolean deleteNamespace(String name) throws ApiException {
 
     if (response.getObject() != null) {
```

Comment:
i know you didn't touch this, but this seems pretty unreliable - we often see issues with stuck namespaces -- we should strengthen this

Reply:
can we create a JIRA to fix this please?